cmd/cephcsi.go

```diff
@@ -70,6 +70,8 @@ func init() {
 	flag.StringVar(&conf.MetricsPath, "metricspath", "/metrics", "path of prometheus endpoint where metrics will be available")
 	flag.DurationVar(&conf.PollTime, "polltime", time.Second*pollTime, "time interval in seconds between each poll")
 	flag.DurationVar(&conf.ProbeTimeout, "timeout", time.Second*probeTimeout, "probe timeout in seconds")
+	flag.StringVar(&conf.HealthzPort, "healthzport", "9808", "TCP ports for listening healthz requests")
```

Comment:
ports are integers, not strings

Reply:
Yeah, my previous comments should answer this.

Reply:
why we need one more port? cant we just add one more endpoint to the metrics port?

Reply:
OK, this is something I followed from the kube liveness probe project, its like giving a provision for users. I'm fine if we just want to use a single port for both metrics and health status.

Check: https://github.com/ceph/ceph-csi/pull/1560/files#diff-e3217918d2c8805d4f5446edf4350a9ee17a8616deb3ce92352a83765c730db5R163